required/latex-lab/latex-lab-math.dtx

```diff
@@ -2470,7 +2470,7 @@
 %    \end{macrocode}
 %    Also for "false?"
 %    \begin{macrocode}
-\def\measuring@true{\let\ifmeasuring@\iftrue\tag_stop:}
+\def\measuring@true{\let\ifmeasuring@\iftrue\tag_suspend:n{\measuring}}
```

Comment:
I thought one \measuring is probably indication enough that we are in a box and that we could save the resume here.

Reply:
I would put it in in the right place, even if commented out for now, so that it it is lear where to uncomment it, but up to you.

Reply:
well actually "the right place" is not trivial, as naturally most is in amsmath directly. So for now I would leave that as it is.

Reply:
but shouldn't all of that be moved to amsmath anyway, eventually? rather than redefining `\measuring@true` here. Okay, doesn't have to happen now, but eventually ...